view/bindings/bindings_test.js

```diff
@@ -24,24 +25,49 @@ steal("can/view/bindings", "can/map", "can/test", "can/view/stache", function (s
 			title: "Sweets",
 			content: "ice cream, candy"
 		}]);
-		var doSomething = function (foodType, el, ev) {
+
+		function doSomething(foodType, el, ev) {
 			ok(true, "doSomething called");
 			equal(el[0].nodeName.toLowerCase(), "p", "this is the element");
 			equal(ev.type, "click", "1st argument is the event");
 			equal(foodType, foodTypes[0], "2nd argument is the 1st foodType");
 
-		};
+		}
 
 		var frag = template({
 			foodTypes: foodTypes,
 			doSomething: doSomething
 		});
 
-		var ta = document.getElementById("qunit-test-area");
 		ta.appendChild(frag);
 		var p0 = ta.getElementsByTagName("p")[0];
 		can.trigger(p0, "click");
 
+
+		template = can.view.mustache("<div>" +
+			"{{#each foodTypes}}" +
+			"<p class='with-args' can-click='{withArgs @event @element title content=content}'>" +
+			"{{content}}</p>" +
+			"{{/each}}" +
+			"</div>");
+		function withArgs(ev1, el1, title, hash, foodType, el2, ev2) {
```

Comment:
Once a user supplies arguments, should the "old defaults" be added?  Should `foodType`, `el2`, `ev2` be passed?

I'm thinking of people checking arguments.length sorta thing.